src/App.vue

```diff
@@ -33,5 +49,6 @@ export default {
   padding: 2em 0 4em;
   flex: 1 1 auto;
   background: var(--color-white);
+  overflow-x: hidden;
```

Comment:
I can imagine this might be confusing in the future, do we need it all the time or just during animation?

Reply:
Well spotted! I've fixed this but setting the overflow during animation only.